packages/headless/src/controllers/folded-result-list/headless-folded-result-list.ts

```diff
@@ -86,6 +89,16 @@ export interface FoldedResultList extends Controller {
    * @param collection - The collection for which to load more results.
    */
   loadCollection(collection: FoldedCollection): void;
```

Comment:
Hum why not bake in the `dispatch(logShowMoreFoldedResults(result))` inside the `loadCollection` function ?

We want to encourage users as much as possible to log analytics on every interaction, and having it automatically done in the function that you need to call anyway to "show more" seems more foolproof.

Having proper analytics recording is a big pain point for any Coveo deployment, and so we must make sure it's as automatic/transparent as possible for implementers, and that they don't have to go out of their way to remember to call a specific function at the right time.

Reply:
makes sense!